initial Dockerfile and docker-compose files (WIP)

jscharber opened the pull request:
WIP

create docker container
docker build -t pavedroadio/prToken:version .

run docker container
docker run -p 8081 -d --name prToken kevlar/prtoken:0.0
-p expose 8081
-d background
--name give it a name

docker stop  prToken

docker-compose up
Starts prToken, cockroach UI, and cockraoch db

docker-compose down stops them

Need more testing and integration with Makefile

jscharber committed:
initial Dockerfile and docker-compose files (WIP)

MarkGreenPR reviewed (approved):
Looks fine

rick4106t reviewed (approved):
is there any significance to the prToken version string being "0.0"?

jscharber committed:
Add support for creating docker image
Add support for starting all services via docker-compose
Add support for converting docker-compose file to k8s
services/deployments with kompose
Update README
Test failed because kevlarweb database was not being created.  Updated
to always create before starting
Added coker-db-only.yaml file for starting only the DB when go test is
executed
Add support for using dep ensure to add and download include deps
Modified .gitignore to not look at sonarcloud scan temp files

jscharber commented:
The latest change for review

See my last commit for more details
